apis/v1/gateway_types.go

```diff
@@ -223,6 +223,8 @@ type GatewaySpec struct {
 	// Implementations MAY merge separate Gateways onto a single set of
 	// Addresses if all Listeners across all Gateways are compatible.
 	//
+	// In a future release the MinItems=1 requirement MAY be dropped.
+	//
```

Comment:
Perhaps this would be better as a Github discussion/issue?

Reply:
@kflynn recommended it here no?

Reply:
Just seemed a bit odd as it seems to be something we would want to track as an eventual change, rather than a reminder in the spec of the potential for that change? Consider this resolved at your discretion.

Reply:
I'll make a github discussion as well.

Reply:
done - https://github.com/kubernetes-sigs/gateway-api/discussions/3636